### ISSUE TITLE
When can we expect to set normalizers using the SimpleFieldAttribute properties?

### ISSUE THREAD

**Issue by dloukola** [Azure/azure-sdk-for-net#24043]
**Query/Question**
When can we expect to set normalizers using the SimpleFieldAttribute properties?
I see that normalizers are in Azure.Search.Documents 11.3.0-beta.1
https://github.com/Azure/azure-sdk-for-net/blob/Azure.Search.Documents_11.3.0/sdk/search/Azure.Search.Documents/CHANGELOG.md
But there doesn't seem to be support for adding the normalizer via an attribute yet.
I expect that a normalizer would be set like any other analyzer. example:
```
[SimpleField(NormalizerName = LexicalNormalizerName.Standard)]
[SearchableField(NormalizerName = LexicalNormalizerName.Standard)]
```
It would be really useful to get a quick answer on IF this is in the backlog and if it is in the backlog, WHEN it would be expected. Otherwise, we'll have to write some workaround for this. Thanks!

**Environment:**
 - Name and version of the Library package used: Azure.Search.Documents 11.3.0-beta.1
 - Hosting platform or OS and .NET runtime version: Windows 10
 - IDE and version : Visual Studio 2019 16.10.4

**Comment by dloukola** [comment]
Any idea if this will be worked on within the next couple of months? otherwise I'll need to figure something else out.

**Comment by Mohit-Chakraborty** [comment]
Hi, We will look into adding this support for our December release of the Search package.
Does that timeline work for you?

**Comment by dloukola** [comment]
Hi @Mohit-Chakraborty , yes Dec works for us. Thank you for your support!

**Comment by dloukola** [comment]
Hi @Mohit-Chakraborty is there any update on this? It's now the end of January and we don't see any normalizer available in the production or even preview versions of the Azure.Search.Documents package.

**Comment by Mohit-Chakraborty** [comment]
We did not release an update in December, team has been focusing on improving the documentation and test coverage. We should have it ready for the March release.

**Comment by dloukola** [comment]
Thank you for the quick reply. We'll have to put a workaround in, but we look forward to the March release.

**Comment by dloukola** [comment]
Should we be able to use normalizers directly without the attributes today? My co-worker tried the solution as shown here:
https://docs.microsoft.com/en-us/rest/api/searchservice/preview-api/test-analyzer
with the REST API and the preview version of the search SDK and neither worked.
Do we have to upgrade our search instance somehow or something?
/cc: @Mohit-Chakraborty

**Comment by Mohit-Chakraborty** [comment]
Adding @dereklegenzoff. We should start by making things work with the raw REST API.

**Comment by pallavit** [comment]
/cc: @alzimmermsft this was a customer ask on the library.

**Comment by ShivangiReja** [comment]
This feature is implemented [here](https://github.com/Azure/azure-sdk-for-net/pull/29568) and released in [11.4.0-beta.9](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/search/Azure.Search.Documents/CHANGELOG.md#1140-beta9-2022-08-08) version.
